packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster-app-card.tsx

```diff
@@ -122,37 +131,64 @@ export const ClusterAppCard: React.FC = () => {
     namespace: undefined,
     name: ALL_APPS,
   });
-  const [pvcSLAData, pvcSLAError, pvcSLALoading] = useCustomPrometheusPoll({
-    endpoint: 'api/v1/query' as any,
-    query: !!cluster ? getPvcSlaPerClusterQuery(cluster) : null,
-    basePath: ACM_ENDPOINT,
+  const [mcvs, mcvsLoaded, mcvsLoadError] = useK8sWatchResource<
+    ACMManagedClusterViewKind[]
+  >({
+    kind: referenceForModel(ACMManagedClusterViewModel),
+    isList: true,
+    namespace: cluster,
+    namespaced: true,
+    optional: true,
     cluster: HUB_CLUSTER_NAME,
   });
+  const [lastSyncTimeData, lastSyncTimeError, lastSyncTimeLoading] =
+    useCustomPrometheusPoll({
+      endpoint: 'api/v1/query' as any,
+      query: LAST_SYNC_TIME_QUERY,
```

Comment:
poll it cluster wise, also send "null" or '' (empty string) for cases when cluster is undefined...
```suggestion
     query: !!cluster ? getLastSyncPerClusterQuery(cluster) : null,
```

Reply:
we can skip the "cluster" part, if this will be hub cluster wise metric with no "cluster" label ??

Reply:
it is hub cluster metric and it has cluster label, we can still pass hub cluster name